vello/src/debug/renderer.rs

```diff
@@ -1,6 +1,10 @@
 // Copyright 2023 the Vello Authors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
+// size_of is not part of the prelude until Rust 1.80 and our MSRV is below that
+#[allow(unused_imports)]
+use core::mem::size_of;
```

Comment:
I wonder whether we should just update MSRV to avoid accidentally having these as (unnecessary) artifacts later, as this isn't used here anyways?

Reply:
The MSRV on `peniko`, `vello`, and everything above it will soon be 1.82. Everything being `masonry` and the Xilem family.

Reply:
Yeah the 1.82 update is coming real soon and I made sure to have all of these as exact copies so it will be easy to remove.

Reply:
> as this isn't used here anyways?

It is used if you check the full PR's diff.

Reply:
The `allow` is necessary because of the `unused_qualifications` Rust compiler lint, which triggers with 1.82 because `size_of` is part of the prelude since 1.80 and thus this explicit import is not needed in 1.82.

The import is still necessary because `size_of` is not yet part of the prelude in 1.76.